db/python/tables/sample.py

```diff
@@ -402,6 +417,12 @@ async def get_sample_with_missing_participants_by_internal_id(
         )
         return {row['id']: row['external_id'] for row in rows}
 
+    async def get_samples_create_date(self, sample_ids: list[int]):
+        """Get a map of {internal_sample_id: date_created} for list of sample_ids"""
+        _query = 'SELECT id, min(row_start) FROM sample FOR SYSTEM_TIME ALL WHERE id in :sids GROUP BY id'
```

Comment:
Just prompting to add this so empty list doesn't cause an exception.

```suggestion
        if len(sample_ids) == 0:
            return []
        _query = 'SELECT id, min(row_start) FROM sample FOR SYSTEM_TIME ALL WHERE id in :sids GROUP BY id'
```

Reply:
Done